app/graphql/types/query_type.rb

```diff
@@ -44,26 +44,35 @@ def record_id(id:, index:)
 
       field :search, SearchType, null: false,
                                  description: 'Search for timdex records' do
-        argument :searchterm, String, required: false, default_value: nil
-        argument :citation, String, required: false, default_value: nil
-        argument :contributors, String, required: false, default_value: nil
-        argument :funding_information, String, required: false, default_value: nil
-        argument :identifiers, String, required: false, default_value: nil
-        argument :locations, String, required: false, default_value: nil
-        argument :subjects, String, required: false, default_value: nil
-        argument :title, String, required: false, default_value: nil
-        argument :from, String, required: false, default_value: '0'
+        argument :searchterm, String, required: false, default_value: nil, description: 'Query all searchable fields'
+        argument :citation, String, required: false, default_value: nil, description: 'Search by citation information'
+        argument :contributors, String, required: false, default_value: nil, description: 'Search by contributor names'
+        argument :funding_information, String, required: false, default_value: nil,
+                                               description: 'Search by funding information; e.g., funding source, award name, etc.'
+        argument :identifiers, String, required: false, default_value: nil,
+                                       description: 'Search by unique indentifier; e.g., ISBN, DOI, etc.'
+        argument :locations, String, required: false, default_value: nil, description: 'Search by locations'
+        argument :subjects, String, required: false, default_value: nil, description: 'Search by subject terms'
+        argument :title, String, required: false, default_value: nil, description: 'Search by title'
+        argument :from, String, required: false, default_value: '0',
+                                description: 'Search result number to begin with (the first result is 0)'
         argument :index, String, required: false, default_value: nil,
                                  description: 'It is not recommended to provide an index value unless we have provided you with one for your specific use case'
 
         # applied facets
-        argument :content_type_facet, [String], required: false, default_value: nil
-        argument :contributors_facet, [String], required: false, default_value: nil
-        argument :format_facet, [String], required: false, default_value: nil
-        argument :languages_facet, [String], required: false, default_value: nil
-        argument :literary_form_facet, String, required: false, default_value: nil
-        argument :source_facet, [String], required: false, default_value: nil
-        argument :subjects_facet, [String], required: false, default_value: nil
+        argument :content_type_facet, [String], required: false, default_value: nil,
```

Comment:
I think I want to rename these arguments (outside the scope of this work but noting it here). We should very likely rename these to replace `_facet` with `_filter`. I'll either open a ticket or include with my deprecation evaluation work.

Reply:
Yeah, `_filter` seems more accurate to me, since they're applied before the search is executed rather than refining the results.